sway/commands/reload.c

```diff
@@ -22,7 +22,9 @@ static void do_reload(void *data) {
 		list_add(bar_ids, strdup(bar->id));
 	}
 
-	if (!load_main_config(config->current_config_path, true, false)) {
+//	if (!load_main_config(config->current_config_path, true, false)) {
```

Comment:
This line should be removed instead of commented out, and I don't think the explanatory comment below it is necessary either.